library/src/test/java/com/datatorrent/lib/io/fs/FileSplitterInputTest.java

```diff
@@ -499,7 +499,7 @@ public void testRecoveryOfBlockMetadataIterator() throws InterruptedException
     testMeta.fileSplitterInput.getScanner().setScanIntervalMillis(500);
 
 
-    testMeta.fileSplitterInput.setup(testMeta.context);
```

Comment:
The goal of writing tests is to test the operator not just setup of window data manager. 
For a trivial improvement, that is, to avoid instantiating objects in the logical instance, this change now doesn't execute setup of the operator which is a much bigger problem. 
In future if something is added to splitter setup, the test will never execute it and result in bugs.

Also, there are multiple of operators in library which instantiate field that are transient when the operator is constructed. Are you going to change all of them?